airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -534,6 +534,7 @@
   dockerRepository: airbyte/source-amazon-ads
   dockerImageTag: 0.1.1
   documentationUrl: https://docs.airbyte.io/integrations/sources/amazon-ads
+  sourceType: api
```

Comment:
I was checking something else and came across that the sourceType is not listed for this so decided to add it anyway as part of this PR